app/deliveryoptions/utils.py

```diff
@@ -0,0 +1,727 @@
+    # PDF or ZIP file - in Discovery, defined in discovery/RDWeb/Services/Mapper/DeliveryOptionsMapper.cs
+    return "(Unknown download format)"
+
+
+def get_download_text(record: dict, surrogate: List) -> str:
+    return "Download now"
+
+
+def get_download_url(record: dict, surrogate: List) -> str:
+    return "details/download"
+
+
+def get_file_authority_type(
+    record: dict, surrogate: List
+) -> (
+    str
+):  # Unknown derivation - EDEV-111. We don't think it is needed, so setting to blank
+    return " "
+
+
+def get_foi_url(record: dict, surrogate: List) -> str:
+    return f"{settings.BASE_TNA_URL}/foirequest?reference={record.reference_number}"
+
+
+def get_image_library_url(record: dict, surrogate: List) -> str:
+    return settings.IMAGE_LIBRARY_URL
+
+
+def get_item_num_of_files_and_size_in_MB(record: dict, surrogate: List) -> str:
+    # On Discovery this is held in Mongo - no equivalent is yet available on Rosetta
+    return "(Unknown number of files and file size)"
+
+
+def get_keepers_gallery_url(record: dict, surrogate: List) -> str:
+    return f"{settings.BASE_TNA_URL}/about/visit-us/whats-on/keepers-gallery/"
+
+
+def get_kew_booking_system_url(record: dict, surrogate: List) -> str:
+    return f"{settings.BASE_TNA_URL}/book-a-reading-room-visit/"
+
+
+def get_max_items(record: dict, surrogate: List) -> str:
+    return settings.MAX_BASKET_ITEMS
+
+
+def get_open_date_desc(record: dict, surrogate: List) -> str:
+    if record.record_opening:
+        return "Opening date: "
+    return " "
+
+
+def get_opening_times_url(record: dict, surrogate: List) -> str:
+    return f"{settings.BASE_TNA_URL}/about/visit-us/"
+
+
+def get_order_url(record: dict, surrogate: List) -> str:
+    # On Discovery, this is related to cookie settings for a DORIS cookie.
+    return "Order URL not yet available"
+
+
+def get_paid_search_url(record: dict, surrogate: List) -> str:
+    return f"{settings.BASE_TNA_URL}/paidsearch/foirequest/{record.iaid}?type=foirequest"
+
+
+def get_price(record: dict, surrogate: List) -> str:
+    # EDEV-109
+    return "(Unknown price)"
+
+
+def get_readers_ticket_url(record: dict, surrogate: List) -> str:
+    return f"{settings.BASE_TNA_URL}/about/visit-us/researching-here/do-i-need-a-readers-ticket/"
+
+
+def get_record_copying_url(record: dict, surrogate: List) -> str:
+    return f"{settings.DISCOVERY_TNA_URL}/pagecheck/start/{record.iaid}/"
+
+
+def get_record_information_type(record: dict, surrogate: List) -> str:
+    # EDEV-110
+    return "(Unknown record information type)"
+
+
+def get_record_opening_date(record: dict, surrogate: List) -> str:
+    if rod := record.record_opening:
+        return rod
+    return " "
+
+
+def get_record_url(record: dict, surrogate: List) -> str:
+    # Subject to change once ds-detail is fleshed out
+    return f"{settings.BASE_TNA_URL}/details/r/{record.iaid}/"
+
+
+def get_first_website_url(record: dict, surrogate: List) -> str:
```

Comment:
Unused record param

Reply:
Closing this as related comment regarding using "_" params in `helpers.py` https://github.com/nationalarchives/ds-search/pull/25/files#r2018153691